### ISSUE TITLE
Add Close/Exit App in the navigation drawer

### ISSUE THREAD

**Issue by cocobelgica** [orgzly/orgzly-android#155]
In most cases, I had to press back multiple times just to exit the app as the back button would take me back to the previous view(Note, Notebooks, Settings) depending on the previous actions I've taken which is kind of tedious when one decides to exit the app.

**Comment by nevenz** [comment]
Any reason you're exiting the app in that way and not just putting it in the background (with home button)? It's not a common thing to have an exit button on Android, as app shouldn't be using much resources (if any) when in the background and system will kill it if it needs memory.

Though navigating through the back stack can be a pain, as Orgzly is keeping every view visited in there. Perhaps we should just make it simple and stop doing that. It would be interesting to know if users even find that behaviour useful.

We could just do what most apps are doing:

- If you click on Notebooks, you end up with just that view in the back stack (pressing back button would close the activity)
- If you click on the notebook, you end up with "Notebooks -> notebook" (pressing back button twice would close the activity)
- We'd still want to be able to go back to search results, but we'd stop keeping different ones etc.

**Comment by cocobelgica** [comment]
Yeah, I could live w/o the Exit/Close button but I think it's more intuitive to keep the back stack simpler just like your suggestion and just like how most apps do. Sometimes - I'm on Notebooks then I hit back and I'm now on Settings(since I was there previously) but in most cases, I was never intending to return there(I would have used the 3-dot menu if it was my intention to go to Settings). Not sure about the other users though.

**Comment by nevenz** [comment]
> Yeah, I could live w/o the Exit/Close button but I think it's more intuitive to keep the back stack simpler just like your suggestion and just like how most apps do.

I think we should just do this. Yes, it's sometimes convenient to get back to the previous notebook (after you've opened two in a row from the drawer), but then again, they are pretty close (swipe & click). And it's not like current position in the notebook you were in is kept (I thought it was, but I just checked).
